backend-nodejs/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Node build artifacts
+node_modules
```

Comment:
I typically like to include a trailing slash on directories whose contents I want ignored.  It's just a visual cue for me to understand "node_modules is a _directory_, and this won't apply to files named "node_modules"